src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -1038,5 +1038,50 @@
     "domain": "www.benefits.va.gov",
     "src": "/gibill/fgib/stem.asp",
     "dest": "/education/other-va-education-benefits/stem-scholarship/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/compensation/agentorange-c123.asp",
+    "dest": "/disability/eligibility/hazardous-materials-exposure/agent-orange/"
+  },
+  {
+    "domain": "www.vets.gov",
+    "src": "/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/diseases/",
```

Comment:
This has a server side redirect- what use case is this for? 

www.vets.gov/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/diseases/